modules/search/pages/create-quick-index.adoc

```diff
@@ -0,0 +1,56 @@
+= Create a Search Index in Quick Mode
+:page-topic-type: guide
+:description: Use Quick Mode to create a Search index in Couchbase Capella with streamlined options.
+
+[abstract]
+{description}
+
+Use xref:create-search-index-ui.adoc[Advanced Mode] to add additional settings and more advanced options for your Search index and xref:customize-index.adoc#type-mappings[type mappings].
+
+You must create a Search index before you can xref:simple-search-ui.adoc[run a search] with the Search Service.
+
+== Prerequisites 
+
+* You have the Search Service enabled on a node in your database.
+For more information about how to change Services on your database, see xref:cloud:clusters:modify-database.adoc[].
+
+* You have a bucket with scopes and collections in your database. 
+For more information, see xref:cloud:clusters:data-service/manage-buckets.adoc[].
+
+* You have logged in to the Couchbase Capella UI. 
+
+== Procedure 
+
+To use Quick Mode in Couchbase Capella to create a Search index:
+
+. On the *Databases* page, select the database where you want to create a Search index.
+. Go to menu:Data Tools[Search].
+. Click btn:[Create Search Index].
++
+By default, *Quick Mode* should be on.
+. In the *Index Name* field, enter a name for the index. 
++
+[NOTE]
+====
+Your index name must start with an alphabetic character (a-z or A-Z). It can only contain alphanumeric characters (a-z, A-Z, or 0-9), hyphens (-), or underscores (_).
+
+For version 7.6 and later, your index name must be unique inside your selected bucket and scope. You cannot have 2 indexes with the same name inside the same bucket and scope.
```

Comment:
Should probably say "For Couchbase Server 7.6 and later"

Reply:
Even though this is Capella? I'm torn on how to refer to the version.